ui/ember-cli-build.js

```diff
@@ -33,9 +18,20 @@ module.exports = function(defaults) {
       plugins: ['transform-object-rest-spread'],
     },
     'ember-cli-babel': {
-      includePolyfill: true,
+      includePolyfill: isProd,
+    },
+    hinting: isTest,
+    tests: isTest,
+    sourcemaps: {
+      enabled: false,
+    },
+    sassOptions: {
+      sourceMap: false,
```

Comment:
This was the thing I wasn't sure if people were using. You _can_ configure your chrome workspace to directly edit SCSS files in the dev tools, but was anyone working that way?